protected/views/audit/_filters.php

```diff
@@ -29,13 +29,12 @@
 			<div class="panel">
 				<div class="row">
 					<div class="large-12 column">
-
 						<table class="grid">
 							<thead>
 								<tr>
-									<th>Site:</th>
-									<th>Firm:</th>
-									<th>User:</th>
+									<th width="15%">Site:</th>
+									<th width="15%">Firm:</th>
+									<th width="20%">User:</th>
```

Comment:
Shouldn't these properties be set through CSS? @marshl I welcome your input here.

Reply:
Agreed, it doesn't look like width is even supported in HTML5 (https://www.w3schools.com/TAGs/tag_th.asp)

Reply:
I'm also not overly keen on using percentages because of unpredictable behaviour on different sized screens. If anyone has an alternative I'd gladly take it on board.